index.html

```diff
@@ -2061,7 +2076,7 @@ <h3>Forms</h3>
             <h2>Protocol Bindings</h2>
             <p>
                 A Protocol Binding is the mapping from an <a>Interaction Affordance</a> to concrete messages of a specific protocol such as HTTP [[!RFC7231]], CoAP [[!RFC7252]], or MQTT [[!MQTT]].
-                It informs the <a>Consumer</a> <em>how</em> to activate the <a>Interaction Affordance</a> through a network-facing interface.
+                It informs the <a>Consumer</a> <em>how</em> to use the <a>Interaction Affordance</a> through a network-facing interface.
```

Comment:
Please revert to activate.
We checked this wording during the hypermedia workshop with IETF people.

Reply:
hmmm. I think the term "activate" is not ideal, since it implies that there is some activity happening at the thing, which is not the case if you read a property.
Using can mean to activate, to read, to subscribe etc, ...

Reply:
"Using" could also just mean to learn what choices exist.

Reply:
"interact with" ?

Reply:
"activate" -- as I said we already invested time into this.
This terminology you can find for instance in texts by Mike Amundsen (the RESTful Web APIs O'Reilly book and more...).